Contentless/Program.cs

```diff
@@ -57,12 +109,20 @@ public static class Program {
             }
         }
 
+        // check references not in .mgcb now
+        foreach (var reference in referencesVersions)
+            if (!referencesSyncs.Contains(reference.Key))
+            {
+                Console.WriteLine($"Please, add reference for {reference.Key} in .mgcb file or remove it from Contentless! Reference was skipped!");
```

Comment:
Can you elaborate on what exactly this message means and/or does? It seems like there should be a way to word this better so that it's clear what it means

Reply:
Sorry, it's a little bit of lazy, In this place I must add new line reference (if tool can't found line in .mgcb), but have decided firstly create PR.